### PR TITLE
Edit title sheager commit

### PR DESCRIPTION
**Pull request by sheager**

### CODE REVIEW

**Comment by SheaPrime** [comment]
comment after pr 23 was closed by sheaprime

**Comment by sheager** [comment]
Reopen PR 23 comment

**Comment by SheaPrime** [comment]
Reopen PR

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 another 1
 stuff
```
Comment (sheager) [inline README.md]: line level change sheager

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
...
 stuffesr
+sheager commit
```
Comment (sheager) [inline README.md]: review conversation #2

Comment (SheaPrime) [reply README.md]: sheaprime commenting on sheager review

**Pull request merged**